### PR TITLE
contrail-vrouter-agent.msi installs contrail-vrouter-api by python pip

### PR DESCRIPTION
**Pull request by kaweue**

### CODE REVIEW

**Review by katrybacka — APPROVED** [review]

**Comment by kaweue** [comment]
Retest this please

File: src/vnsw/agent/contrail/windows/msi/agent_msi.wxs [hunk 1]
```diff
@@ -27,6 +27,9 @@
                             <ServiceInstall Id="CreateService" Name="ContrailAgent" DisplayName="ContrailAgent" Type="ownProcess" Start="demand" ErrorControl="normal"/>
                             <ServiceControl Id="DeleteService" Name="ContrailAgent" Remove="both" Stop="both" Wait="yes"/>
                         </Component>
+                         <Component Id="ContrailVrouterApi" Guid="21F50C2B-7F38-493B-B545-EC7DED2CB2F7" Win64="yes">
```
Comment (krdln) [inline src/vnsw/agent/contrail/windows/msi/agent_msi.wxs]: one space too much

File: src/vnsw/agent/contrail/windows/msi/agent_msi.wxs [hunk 2]
```diff
@@ -69,5 +73,18 @@
             <ComponentRef Id="SampleConfig"/>
             <ComponentRef Id="ContrailVrouterAgentConfig"/>
         </Feature>
+
+        <CustomAction Id="InstallContrailVrouterApi" Property="InstallContrailVrouterApiProperty" Value="&quot;pip2.exe&quot; install --upgrade &quot;[#ContrailVrouterApi]&quot;" Execute="immediate"/>
```
Comment (krdln) [inline src/vnsw/agent/contrail/windows/msi/agent_msi.wxs:77]: Are these `&quot;`s necessary? Escpecially around `pip2.exe`

File: src/vnsw/agent/contrail/windows/msi/agent_msi.wxs [hunk 3]
```diff
@@ -27,6 +27,9 @@
                             <ServiceInstall Id="CreateService" Name="ContrailAgent" DisplayName="ContrailAgent" Type="ownProcess" Start="demand" ErrorControl="normal"/>
                             <ServiceControl Id="DeleteService" Name="ContrailAgent" Remove="both" Stop="both" Wait="yes"/>
                         </Component>
+                         <Component Id="ContrailVrouterApi" Guid="21F50C2B-7F38-493B-B545-EC7DED2CB2F7" Win64="yes">
+                            <File Id="ContrailVrouterApi" Source="build\noarch\contrail-vrouter-api\dist\contrail-vrouter-api-1.0.tar.gz"/>
```
Comment (krdln) [inline src/vnsw/agent/contrail/windows/msi/agent_msi.wxs]: should it be copied here for sure?

**Review by krdln — CHANGES_REQUESTED** [review]

File: src/vnsw/agent/contrail/windows/msi/agent_msi.wxs (hunk 2 above)
Comment (kaweue) [reply src/vnsw/agent/contrail/windows/msi/agent_msi.wxs:77]: Yes it is

File: src/vnsw/agent/contrail/windows/msi/agent_msi.wxs (hunk 3 above)
Comment (kaweue) [reply src/vnsw/agent/contrail/windows/msi/agent_msi.wxs]: I will check if it is easy to do.

Comment (kaweue) [reply src/vnsw/agent/contrail/windows/msi/agent_msi.wxs]: And use '/' instead of '\\'

Comment (kaweue) [reply src/vnsw/agent/contrail/windows/msi/agent_msi.wxs]: Unfortunately WiX doesn't have straightforward solution to use this file from temporary directory.

**Review by krdln — APPROVED** [review]

**Pull request merged**